moabb/tests/contexts.py

```diff
@@ -26,12 +23,10 @@ class Test_CrossSess(unittest.TestCase):
     run it. Putting this on the future docket...
 
     '''
-    def tearDown(self):
-        if os.path.isfile('results.hd5'):
-            os.remove('results.hd5')
```

Comment:
Why remove the tearDown? We want the runs to work independently right?

Reply:
it only call eval, so no results are generated here.

Reply:
ah gotcha